tutor/resources/styles/components/new-course.less

```diff
@@ -1,140 +1,337 @@
-.new-course {
-  max-width: 400px;
-  margin: 0 auto;
+.psuedo-background() {
+  position: absolute;
+  top: 0;
+  left: 0;
+  bottom: 0;
+  right: 0;
+  content: "";
+  z-index: -1;
+  background-repeat: no-repeat;
+  background-size: cover;
+}
 
-  &::before {
-    position: absolute;
-    top: 0;
-    left: 0;
-    bottom: 0;
-    right: 0;
-    .tutor-background-image('ost-hero.jpg');
-    opacity: .4;
-    content: "";
-    z-index: -1;
-    background-repeat: no-repeat;
-    background-size: cover;
-  }
+.make-wizard-choice-active() {
+  background-color: @tutor-neutral-light;
+  border-color: @tutor-neutral-light;
+  color: @tutor-neutral-dark;
+}
 
-  .panel {
-    .panel-heading,
-    .panel-footer {
-      .controls { text-align: right; }
-      color: @tutor-neutral-dark;
-      background-color: white;
-      border: 0;
+.new-course-wizard {
+  .list-group-item {
+    padding: 8px;
+
+    &:focus {
+      outline-color: @tutor-neutral;
+      outline-width: thin;
+      outline-style: dotted;
     }
-    .panel-heading {
-      font-size: 1.8rem;
-      margin-top: 1.8rem;
-      padding-bottom: 0;
-    }
-  }
-  tbody tr {
-    cursor: pointer;
-    &.selected {
-      background-color: @tutor-neutral-light;
-      td:last-child {
-        position: relative;
-        &:after {
-          .fa-icon();
-          content: @fa-var-check;
-          color: @tutor-primary;
-          position: absolute;
-          top: calc(~"50% - 0.8rem");
-          font-size: 1.8rem;
-          right: 5px;
+
+    &.active {
+      .make-wizard-choice-active();
+
+      &:hover,
+      &:focus {
+        .make-wizard-choice-active();
+      }
+
+      .course-choice {
+        &::after {
+          top: 16px;
+          width: 28px;
+          height: 28px;
+          content: '';
+          background-image: url('../../images/icons/icon-checked.svg');
+          background-color: @tutor-white;
+          background-repeat: no-repeat;
+          background-position: center center;
+          display: table-cell;
+          vertical-align: middle;
+          border-radius: 50%;
         }
       }
     }
-  }
 
-  .select-type {
+    &[data-brand] {
+      .course-choice-content-wrapper::before {
+        position: absolute;
+        top: 10px;
+      }
+    }
+
+    &[data-brand=coach] {
+      .course-choice-content-wrapper::before {
+        content: url('../../images/openstax-logo-coach-horizontal.svg');
+      }
+    }
 
-    display: flex;
-    border: 1px solid @tutor-neutral-light;
+    &[data-brand=tutor-beta] {
+      .course-choice-content-wrapper::before {
+        content: url('../../images/openstax-logo-tutor-beta-horizontal.svg');
+      }
+    }
 
```

Comment:
Just curious but why the "-beta" ?  Seems like something that shouldn't be embedded in the code since now we should go through the codebase and remove it once it's "production".  It's not a big deal though since there's also user-visible labels that'll need updating as well.